vllm/model_executor/layers/fused_moe/modular_kernel.py

```diff
@@ -509,6 +528,9 @@ class FusedMoEModularKernel(torch.nn.Module):
     layer due to any layer specific state that may be used by the component
     objects.
     """
+    fused_out_buffer = SharedResizableBuffer()
+    workspace13_buffer = SharedResizableBuffer()
+    workspace2_buffer = SharedResizableBuffer()
```

Comment:
Why do we need this?

Reply:
Just a general memory footprint reduction. Primarily targeting cudagraphs, though.